### PR TITLE
fix(bugfix/fix-footer): update footer

### PR DESCRIPTION
**Pull request by HoangLiarPhuongLe**
- Delete `padding left` and `padding right` of class `.footer-section` in file footer.css
- Setup width and margin for class `.footer-container` in file footer.css
- Merge two class `.footer-item-top` and `.footer-item-bottom` become class` .footer-item` for avoid duplicate code
- Rename class` .rule` become class `.footer-rule`
- Rounding number the value of `opacity`
- Split and add a comment for each part as font, color,... of `:root{ }`
- Add `text-transform: uppercase` for class `navbar-item ` 
- Delete `a:hover { }`
- Support clickable email and tel number

### CODE REVIEW

File: practice-one/src/index.html [hunk 1]
```diff
@@ -4,10 +4,10 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Document</title>
+    <link rel="stylesheet" href="assets/css/reset.css" type="text/css"/>
     <link rel="stylesheet" href="assets/css/global.css" type="text/css"/>
     <link rel="stylesheet" href="assets/css/header.css" type="text/css"/>
     <link rel="stylesheet" href="assets/css/footer.css" type="text/css"/>
-    <link rel="stylesheet" href="assets/css/reset.css" type="text/css"/>
     <link rel="stylesheet" href="assets/css/text.css" type="text/css"/>
```
Comment (ghost) [inline practice-one/src/index.html:11]: @HoangLiarPhuongLe  Is it appropriate to import text styles after the header and footer?

Comment (ghost) [reply practice-one/src/index.html:11]: @HoangLiarPhuongLe In this file, are you only styling text? If not, why is it named `text.css`?

**Review by ghost — COMMENTED** [review]
Screenshot 2023-07-11 150604

@HoangLiarPhuongLe 
`css` is a file extension commonly used for files containing CSS code. Naming a folder as "css" can create confusion because the folder name matches the file extension.

Instead, you can choose a more appropriate folder name, for example: "styles". By doing so, you can include not only CSS files but also other CSS-related formats like SCSS, LESS, or any other CSS code formats you may use in the future.

File: practice-one/src/assets/css/footer.css [hunk 2]
```diff
@@ -1,23 +1,18 @@
 .footer-section {
   background-color: var(--black-color);
   padding-top: 105px;
-  padding-left: 147px;
-  padding-right: 151px;
   padding-bottom: 91px;
 }
+
 .footer-container {
```
Comment (phuongbuimai-agilityio) [inline practice-one/src/assets/css/footer.css:7]: The container class is used to wrap the content of all sections in a web. It should be a standard class.

File: practice-one/src/assets/css/footer.css [hunk 3]
```diff
@@ -35,26 +30,31 @@
...
   max-height: 77px;
 }
+
 .break-about-us {
   max-width: 360px;
   max-height: 48px;
 }
-.rule {
+
+.footer-rule {
```
Comment (phuongbuimai-agilityio) [inline practice-one/src/assets/css/footer.css:49]: Why do we need this class? Do we have another way to make the line in the footer?

File: practice-one/src/index.html [hunk 4]
```diff
@@ -78,29 +82,29 @@ <h4 class="text-primary text-white text-title-footer">about us</h4>
...
+            <hr class="footer-rule">
 
-            <div class="footer-item-bottom">
+            <div class="footer-item">
                 <div class="text-primary description text-white opacity">© 2023 SmartWeb</div>
                 <nav>
                     <ul class="navbar-menu">
                         <li>
-                            <a class="text-primary navbar-item text-white" href="#">HOME</a>                    
+                            <a class="text-primary navbar-item text-white" href="#">home</a>
```
Comment (phuongbuimai-agilityio) [inline practice-one/src/index.html:92]: Duplicate code

File: practice-one/src/assets/css/footer.css [hunk 5]
```diff
@@ -35,26 +30,31 @@
...
   width: 100%;
   height: 1px;
   background-color: var(--white-color);
   border-style: solid;
   margin-top: 42px;
   margin-bottom: 38px;
 }
+
 .opacity {
```
Comment (phuongbuimai-agilityio) [inline practice-one/src/assets/css/footer.css:58]: As I told you, why don't you set this property for the class of the item?

**Pull request closed without merging**